builtins/amp-ad.js

```diff
@@ -49,6 +49,11 @@ export function installAd(win) {
 
   class AmpAd extends BaseElement {
 
+    /** @override */
+    getElementPriority() {
```

Comment:
I think this should be called `getPriority`. Otherwise we repeat the string from the class name in the method name.